pep-0655.rst

```diff
@@ -315,7 +315,7 @@ Usage in Python <3.11
 If your code supports Python <3.11 and wishes to use ``Required[]`` or
 ``NotRequired[]`` then it should use ``typing_extensions.TypedDict`` rather
 than ``typing.TypedDict`` because the latter will not understand
-``(Not)Required[]``. In particular ``__required_keys__`` and 
+``(Not)Required[]``. In particular ``__required_keys__`` and
```

Comment:
It's not clear to me why these extra formatting changes are here. Perhaps your editor is autoformatting whitespace?

Reply:
(Ditto for additional formatting changes below in the diff.)

Reply:
Yeah, my editor automatically cleans up stray trailing whitespace on save, which it looks like this PEP had. I didn't see anything else in flight touching on this PEP and expected a fairly quick merge, so when I noticed it I didn't immediately go back and rewrite the commit without the whitespace changes. But if you are currently working on additional changes that this might cause a merge conflict with, I can go ahead and do that—just let me know.